expression/integration_test.go

```diff
@@ -850,8 +850,8 @@ func (s *testIntegrationSuite) TestStringBuiltin(c *C) {
 	result.Check(testkit.Rows("'121' '0' '中文' <nil>"))
 
 	// for convert
-	result = tk.MustQuery(`select convert("中文" using "utf8"), convert(cast("中文" as binary) using "utf8");`)
-	result.Check(testkit.Rows("中文 中文"))
+	result = tk.MustQuery(`select convert("123" using "binary"), convert("中文" using "binary"), convert("中文" using "utf8"), convert(cast("中文" as binary) using "utf8");`)
+	result.Check(testkit.Rows("123 中文 中文 中文"))
```

Comment:
What about `select convert("123" using "866");` ?

Reply:
@winkyao TiDB supports "866",  I add a test for it.  

Or should we delete charset "866" because MySQL doesn't support "866" ？